.storybook/webpack.config.js

```diff
@@ -33,9 +33,6 @@ module.exports = (storybookBaseConfig, configType) => {
         // improves compile time on larger projects
         Object.assign(
           {},
-          {
-            loader: require.resolve('thread-loader'),
-          },
```

Comment:
Turns out builds were faster when not using threads.

Reply:
Really?

Reply:
Yes, really. Maybe it depends on the machine, but for me it was faster. See the `thread-loader` section in the PR description.